src/paper/management/commands/backfill_concepts.py

```diff
@@ -0,0 +1,76 @@
+from datetime import datetime
+
+from django.core.management.base import BaseCommand
+
+from paper.related_models.paper_model import Paper
+from tag.models import Concept
+from utils.openalex import OpenAlex
+
+
+class Command(BaseCommand):
+    help = "Backfill Concepts for Papers"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--start_date", type=str, help="Start date in YYYY-MM-DD format."
+        )
+        parser.add_argument("--doi", type=str, help="DOI of a specific paper.")
+
+    def handle(self, *args, **kwargs):
+        start_date_str = kwargs["start_date"]
+        doi = kwargs["doi"]
+        open_alex = OpenAlex()
+
+        if start_date_str:
+            start_date = datetime.strptime(start_date_str, "%Y-%m-%d").date()
+            papers = Paper.objects.filter(created_date__gte=start_date)
+            print(f"Found {papers.count()} papers created after {start_date_str}")
+        elif doi:
+            papers = Paper.objects.filter(doi=doi)
+        else:
+            self.stdout.write(self.style.ERROR("Please provide a start date or DOI."))
+            return
+
+        for paper in papers:
+            try:
```

Comment:
Can we also print here how many are left to do? I think this will probably take a long time, so knowing how many entries are left would be good